src/api/utils/notifications/README.md

```diff
@@ -0,0 +1,55 @@
+# Andela Societies Notifications
+## About
+This module implements notifications for the Andela Societies Platform.
+## Goal
+The goal of this project is to implement notifications asynchronously in the Andela Societies Platform.
+## Features
+With this module;
+- You can send email notifications using MailGun.
+- TODO: You can send Slack notifications.
+- TODO: You can send Telegram notifications.
+## Requirements
+- Use Python 3.6.5
+- Use Flask 0.12.2
+## Tests
+"Code without tests is broken as designed", said  [Jacob Kaplan-Moss](https://jacobian.org/writing/django-apps-with-buildout/#s-create-a-test-wrapper). Therefore I shall not give you code that
+can not be tested or has no tests. So, to run tests, enter the following command
+```sh
+    $ python src/manage.py tests
+```
+## Usage
+To use this module:
+
+- You need to install [rabbitMQ server](https://www.rabbitmq.com/install-standalone-mac.html) (broker)
+- You need to install [celery](http://docs.celeryproject.org/en/latest/getting-started/first-steps-with-celery.html) (job server)
+Then issue the following commands
+```sh
+    $ brew services start rabbitmq
+    $ celery -A src.api.utils.notifications.email_notices worker
+``
+You need to set the following environment variables:
+```bash
+export MAIL_GUN_URL=https://api.mailgun.net/v3/sandbox1ce36b0401fd4c929ee864bc19bf44e7.mailgun.org/messages
+export MAIL_GUN_API_KEY=fef6a810b6acceac40cf2fcb1ebac4fe-0470a1f7-a03c7123
+export SENDER_CREDS="Andela Societies Test  <postmaster@sandbox1ce36b0401fd4c929ee864bc19bf44e7.mailgun.org>"
+```
+Then in your module/controller do the following:
+
+```python
+# import the notifications
+from src.api.utils.notifications.email_notices import send_email
+
+...
+
+class SomeEndPoint(Resource):
+    def post(self):
+        # some other business logic here
+        ...
+        # At this point you need to send the email
+        send_email.delay(
+            subject="Test email",
+            message="This is a test message",
+            recipients=["andela.project@gmail.com"],
+            sender="Mailgun Sandbox <postmaster@sandbox1ce36b0401fd4c929ee864bc19bf44e7.mailgun.org>"
+        )
+```
```

Comment:
This looks like extra documentation, not related to set-up, you can add it to the repo's WIKI rather than in the readme.